tests/Parsing/ParsingTest.elm

```diff
@@ -0,0 +1,98 @@
+module Parsing.ParsingTest exposing (parseTest)
+
+import Elm.Parser
+import Elm.Processing
+import Elm.Syntax.File
+import Elm.Syntax.Range
+import Expect
+import Intermediate
+import Models
+import Test
+import TestUtil
+
+
+parseTest : Test.Test
+parseTest =
+    let
+        checkExpectation expected modString =
+            case TestUtil.stringToIntermediate modString of
+                Just parsed ->
+                    Expect.equal parsed expected
+
+                Nothing ->
+                    Expect.fail "Failed to parse the module!"
+    in
+    Test.describe "Test the parsing function in its entirety."
+        [ Test.test "Empty module." <|
+            \() ->
+                checkExpectation
+                    { entities = []
+                    , moduleName = "SomeName"
+                    , topLevelComment = Nothing
+                    , otherComments = []
+                    }
+                    "module SomeName exposing (..)"
+        , Test.test "Large example." <|
```

Comment:
Similarly to the comment above -- why is this example "large" (and what would be a "small" example)? The reader can already see that it is "large" :) But what is the essence of this "large" -- you test multiple entities? You test that it can handle strings larger than x characters?